cekit/tools.py

```diff
@@ -647,18 +647,11 @@ def handle(self, o: Any, params: Map) -> None:
         if not o:
             return
 
-        # Get the class of the object
-        clazz = type(o)
+        if callable(getattr(o, "dependencies", None)):
```

Comment:
I think this should be fine. I briefly checked and we don't have any static methods on which we would need to execute it.

Reply:
It does actually call e.g.:
```
    @staticmethod
    def dependencies(params: Map = None, image: Image = None) -> DependencyDefinition:
```

but brief testing seems to imply getattr works for static and non-static